### PR TITLE
network: Introduce force renew for DHCP server

### PR DESCRIPTION
**Pull request by ssahani**
We already have FORCERENEW support in DHCP server. Extend it to networkctl and bus method and make is usable.

### CODE REVIEW

File: src/network/networkctl.c [hunk 1]
```diff
@@ -2145,14 +2188,15 @@ static int parse_argv(int argc, char *argv[]) {
...
-                { "reload",      1,        1,        0,            verb_reload         },
+                { "list",            VERB_ANY, VERB_ANY, VERB_DEFAULT, list_links          },
+                { "status",          VERB_ANY, VERB_ANY, 0,            link_status         },
+                { "lldp",            VERB_ANY, VERB_ANY, 0,            link_lldp_status    },
+                { "label",           VERB_ANY, VERB_ANY, 0,            list_address_labels },
+                { "delete",          2,        VERB_ANY, 0,            link_delete         },
+                { "renew",           2,        VERB_ANY, 0,            link_renew          },
+                { "forcerenew",      2,        VERB_ANY, 0,            link_force_renew    },
+                { "reconfigure",     2,        VERB_ANY, 0,            verb_reconfigure    },
+                { "reload",          1,        1,        0,            verb_reload         },
```
Comment (yuwata) [inline src/network/networkctl.c]: Why do you reindent them? It seems not necessary.

File: man/networkctl.xml [hunk 2]
```diff
@@ -267,6 +267,14 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
         Takes interface name or index number.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>
+          <command>forcerenew</command>
+        </term>
+        <listitem><para>Force nenew the  DHCP server configuration to all the clients.
```
Comment (yuwata) [inline man/networkctl.xml]: duplicated white space.

File: src/network/networkctl.c (hunk 1 above)
Comment (ssahani) [reply src/network/networkctl.c]: Ouch. first I named this to dhcp-forcerenew. Then I forgot in reindent.

**Comment by ssahani** [comment]
Updated thanks for review @yuwata . PTAL

**Comment by yuwata** [comment]
LGTM.

File: src/network/networkctl.c [hunk 3]
```diff
@@ -2050,6 +2092,7 @@ static int help(void) {
                "  label                  Show current address label entries in the kernel\n"
                "  delete DEVICES...      Delete virtual netdevs\n"
                "  renew DEVICES...       Renew dynamic configurations\n"
+               "  force-renew DEVICES... DHCP server sends a FORCERENEW configurations\n"
```
Comment (dnicolodi) [inline src/network/networkctl.c]: `s/force-renew/forcerenew/`, `s/configurations/configuration/`. I think the command description can be improved: "Trigger DHCP reconfiguration of all connected clients"

File: man/networkctl.xml [hunk 4]
```diff
@@ -267,6 +267,14 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
         Takes interface name or index number.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>
+          <command>forcerenew</command>
+        </term>
+        <listitem><para>Force nenew the DHCP server configuration to all the clients.
```
Comment (dnicolodi) [inline man/networkctl.xml]: Typo: `s/nenew/renew/`. Something like "Send a FORCERENEW message to all connected clients, triggering DHCP reconfiguration." may be a better.

**Pull request merged**